src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2819,8 +2844,12 @@ private uint getJitFlags(ref CORJIT_FLAGS flags, uint sizeInBytes)
             flags.Set(CorJitFlag.CORJIT_FLAG_PREJIT);
             flags.Set(CorJitFlag.CORJIT_FLAG_USE_PINVOKE_HELPERS);
 
-            if ((_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.X86
-                || _compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.X64)
+            TargetArchitecture targetArchitecture = _compilation.TypeSystemContext.Target.Architecture;
+            if (targetArchitecture == TargetArchitecture.ARM)
+                flags.Set(CorJitFlag.CORJIT_FLAG_RELATIVE_CODE_RELOCS);
```

Comment:
The unmanaged crossgen does this for Unix ARM only.

Reply:
Thank you. Fixed.